src/components/footer-list/FooterList.jsx

```diff
@@ -1,27 +1,19 @@
 import { Box } from "@mui/material"
 import { List, ListItemButton, ListSubheader, ListItemText } from '@mui/material';
+import styles from "./FooterList.module.css";
 
-export const FooterList = ({ title, dataArr, type }) => {
-    const styles = {
-        lineHeight: "18px",
-        fontSize: 14,
-        fontWeight: 500,
-        textTransform: "capitalize",
-        paddingInline: "0px",
-        paddingBlock: "4px",
-        width: "100%"
-    }
+export const FooterList = ({ title, dataSource, type }) => {
     return (
         <Box sx={{ width: "fit-content" }}>
             <List>
-                <ListSubheader style={styles} sx={{ color: "#fff", background: "transparent", marginBottom: "8px" }}>
+                <ListSubheader className={styles.commenTextStyle} sx={{ color: "#fff", background: "transparent", marginBottom: "8px" }}>
```

Comment:
again bad naming find a more suitable name like footerTextStyle, this way I know this style is for the footer text, naming things in programing is a very important skill